experiments/openebs/openebs-pool-disk-loss/disk_status_check.j2

```diff
@@ -0,0 +1,32 @@
+# All code here is not indented because j2 is space sensitive
```

Comment:
this is common for all disk loss, can we have it in a lib folder. this would cause a change in other disk loss (generic, kafka) - but would be worth doing.